projects/demo-integrations/cypress/tests/desktop/addon-doc/navigation.cy.ts

```diff
@@ -1,4 +1,4 @@
-import {WAIT_BEFORE_SCREENSHOT} from '../../support/shared.entities';
+import {WAIT_BEFORE_SCREENSHOT} from '../../../support/shared.entities';
```

Comment:
`import {WAIT_BEFORE_SCREENSHOT} from '@demo-integrations/support/shared.entities';`

add new `path` to `paths` properties in tsconfig.json